src/StockAvailabilityChecker.php

```diff
@@ -51,15 +51,17 @@ public function check(PurchasableEntityInterface $entity, $quantity, Context $co
     }
```

Comment:
ok I can see that but lets push it in a cleanup/general commit. I want the commit to show what needed changing for this to work. 
For me its meaningful to be able to easily tell that the location as entity did not change the StockAvailabilityChecker.  
No need to update the PR. will do it in the commit.